`return_columns=True` for `FastCRUD.update(...)` errors in `TypeError: 'bool' object is not iterable`

**Description**
I was working on a patch to allow polymorphic inheritance. I stumbled upon `fastcrud.crud.fast_crud::FastCRUD.update(...)` with `return_columns=True`.

Inside of `update()`, there is a call of `fastcrud.crud.execution.execute_update_and_return_response()` that doesn't handle `return_columns=True`:
```python
    if return_columns:
        stmt = stmt.returning(*[column(name) for name in return_columns])
        db_row = await db.execute(stmt)
        if commit:
            await db.commit()

        if allow_multiple:
            multi_data = [dict(row) for row in db_row.mappings()]
            formatted_data = format_multi_response(
                multi_data, schema_to_select, return_as_model
            )
            return {"data": formatted_data}
```
and I get an error:
```
>           stmt = stmt.returning(*[column(name) for name in return_columns])
                                                             ^^^^^^^^^^^^^^
E           TypeError: 'bool' object is not iterable
```

However, `update()`'s docstring says
```
return_columns: A list of column names to return after the update. If `return_as_model` is True, all columns are returned.
```

**To Reproduce**
```python
import pytest

from fastcrud import FastCRUD
from pydantic import BaseModel

# conftest.py...
class Entity(Base):
    __tablename__ = 'entities'
    __mapper_args__ = {
        'polymorphic_on': 'type',
        'polymorphic_identity': 'entity',
        'with_polymorphic': '*',
    }
    id = Column(Integer, primary_key=True, autoincrement=True)
    type = Column(String(100), nullable=True, default='entity')


class UserEntity(Entity):
    __tablename__ = 'users_entities'
    __mapper_args__ = {
        'polymorphic_identity': 'user_entity',
    }
    id = Column(ForeignKey('entities.id'), primary_key=True)
    name = Column(String(100), nullable=True)


class UserEntityCreateSchema(BaseModel):
    name: str


# SQLAlchemy test...
@pytest.mark.asyncio
async def test_update_polymorphic(async_session):

    user = UserEntity(name="User A")
    async_session.add(user)
    await async_session.commit()
    await async_session.refresh(user)

    new_user_name = "Update User Name"
    new_raw_data = UserEntityCreateSchema(name=new_user_name)

    crud = FastCRUD(model=UserEntity)
    fetched_record = await crud.update(
        db=async_session,
        object=new_raw_data,
        return_as_db_obj=True,
        return_columns=True,
        id=user.id,
    )

    assert fetched_record is not None
    assert fetched_record.id == 1
    assert fetched_record.name == new_user_name
    assert fetched_record.type == "user_entity"


```

--- a/fastcrud/crud/fast_crud.py
+++ b/fastcrud/crud/fast_crud.py
@@ -2881,6 +2881,11 @@ class FastCRUD(
             )
             ```
         """
+        if return_columns is not None and not isinstance(return_columns, list):
+            raise ValueError(
+                "return_columns must be a list of column name strings or None."
+            )
+
         await validate_update_delete_operation(
             self.count, db, allow_multiple, "update", **kwargs
         )
@@ -2890,11 +2895,6 @@ class FastCRUD(
 
         filters = self._filter_processor.parse_filters(**kwargs)
         stmt = update(self.model).filter(*filters).values(update_data)
-
-        if return_columns is not None and not isinstance(return_columns, list):
-            raise ValueError(
-                "return_columns must be a list of column name strings or None."
-            )
 
         if return_as_model:
             return_columns = self.model_col_names

--- a/fastcrud/crud/fast_crud.py
+++ b/fastcrud/crud/fast_crud.py
@@ -2891,6 +2891,11 @@ class FastCRUD(
         filters = self._filter_processor.parse_filters(**kwargs)
         stmt = update(self.model).filter(*filters).values(update_data)
 
+        if return_columns is not None and not isinstance(return_columns, list):
+            raise ValueError(
+                "return_columns must be a list of column name strings or None."
+            )
+
         if return_as_model:
             return_columns = self.model_col_names
 

--- a/tests/sqlalchemy/crud/test_update.py
+++ b/tests/sqlalchemy/crud/test_update.py
@@ -183,9 +183,9 @@ async def test_update_with_schema_object(async_session, test_data):
         select(ModelTest).where(ModelTest.id == target_id)
     )
     updated = updated_record.scalar_one()
-    assert (
-        updated.name == "Updated Via Schema Object"
-    ), "Record should be updated with the name from the schema object."
+    assert updated.name == "Updated Via Schema Object", (
+        "Record should be updated with the name from the schema object."
+    )
 
 
 @pytest.mark.asyncio
@@ -207,9 +207,9 @@ async def test_update_auto_updates_updated_at(async_session, test_data):
     )
     updated = updated_record.scalar_one()
     assert updated.name == "UpdatedName", "Record should be updated with the new name."
-    assert (
-        updated.updated_at > initial_time
-    ), "updated_at should be later than the initial timestamp."
+    assert updated.updated_at > initial_time, (
+        "updated_at should be later than the initial timestamp."
+    )
 
 
 @pytest.mark.parametrize(
@@ -275,3 +275,19 @@ async def test_update_with_returning(
     # Rollback the current transaction to see if the record was actually committed
     await async_session.rollback()
     assert await crud.count(async_session, name="Updated Name") == 1
+
+
+@pytest.mark.asyncio
+async def test_update_return_columns_bool_raises_value_error(async_session, test_data):
+    for item in test_data:
+        async_session.add(ModelTest(**item))
+    await async_session.commit()
+
+    crud = FastCRUD(ModelTest)
+    with pytest.raises(ValueError, match="return_columns must be a list"):
+        await crud.update(
+            db=async_session,
+            object={"name": "Updated"},
+            return_columns=True,
+            id=test_data[0]["id"],
+        )

--- a/tests/sqlalchemy/crud/test_update.py
+++ b/tests/sqlalchemy/crud/test_update.py
@@ -183,9 +183,9 @@ async def test_update_with_schema_object(async_session, test_data):
         select(ModelTest).where(ModelTest.id == target_id)
     )
     updated = updated_record.scalar_one()
-    assert updated.name == "Updated Via Schema Object", (
-        "Record should be updated with the name from the schema object."
-    )
+    assert (
+        updated.name == "Updated Via Schema Object"
+    ), "Record should be updated with the name from the schema object."
 
 
 @pytest.mark.asyncio
@@ -207,9 +207,9 @@ async def test_update_auto_updates_updated_at(async_session, test_data):
     )
     updated = updated_record.scalar_one()
     assert updated.name == "UpdatedName", "Record should be updated with the new name."
-    assert updated.updated_at > initial_time, (
-        "updated_at should be later than the initial timestamp."
-    )
+    assert (
+        updated.updated_at > initial_time
+    ), "updated_at should be later than the initial timestamp."
 
 
 @pytest.mark.parametrize(

--- a/tests/sqlalchemy/endpoint/test_cursor_validation.py
+++ b/tests/sqlalchemy/endpoint/test_cursor_validation.py
@@ -9,6 +9,7 @@ import pytest
 from typing import Annotated
 from fastapi import FastAPI, Depends
 from fastapi.testclient import TestClient
+from pydantic import BaseModel
 from sqlalchemy import Column, Integer, BigInteger, SmallInteger, String, DateTime
 from sqlalchemy.dialects.postgresql import UUID as PostgresUUID
 from sqlalchemy.orm import declarative_base
@@ -32,18 +33,18 @@ class TestCursorModel(Base):
     uuid_field = Column(PostgresUUID(as_uuid=True))
 
 
+class CreateSchema(BaseModel):
+    name: str
+
+
+class UpdateSchema(BaseModel):
+    name: str
+
+
 @pytest.fixture
 def app_with_cursor_validation(async_session):
     """Create a FastAPI app with cursor validation."""
     app = FastAPI()
-
-    from pydantic import BaseModel
-
-    class CreateSchema(BaseModel):
-        name: str
-
-    class UpdateSchema(BaseModel):
-        name: str
 
     endpoint_creator = EndpointCreator(
         session=lambda: async_session,

